civo/database/resource_database_test.go

```diff
@@ -44,33 +46,34 @@ func TestAccCivoDatabase_basic(t *testing.T) {
 	})
 }
 
-func TestAccCivoDatabase_update(t *testing.T) {
+// CivoDatabase_update is used to test the database resource
+func CivoDatabase_update(t *testing.T) {
```

Comment:
Why not calling `Test<FunctionUnderTest>` the test?

Reply:
I will check